### DISCUSSION TITLE
Contributions?

### DISCUSSION THREAD

**Discussion by Quan1umMango** [AryanpurTech/BlueEngine#29] [Q&A]
Hey , I was just wondering if there's a contribution guide or if contributing is available.

**Comment by ElhamAryanpur** [comment]
Oh hello, apologies I haven't seen this discussion before.

There is no contribution guide yet, but you can write articles, make videos, or generally use the engine and report problems or feature requests. Those are the most valuable contributions you can make at this point!

Generally just tutorials and guides are way more than enough as we lack them heavily.

Thank you!
